javascript/code-challenges/multiBracketValidation/README.md

```diff
@@ -0,0 +1,27 @@
+# Multi Bracket Validation
+
+Test whether the brackets in a string are balanced
+Input: a string as its only argument
+Output:a boolean representing whether or not the brackets in the string are balanced.
+
+There are 3 types of brackets:
+
+    Round Brackets : ()
+    Square Brackets : []
+    Curly Brackets : {}
+
+## Challenge
+
+
+
+## Approach & Efficiency
+
+### Big O Notation
+
+- 
+
+## Testing
+
+## Whiteboard Solution
+
+![code-challenge-13](./assets/CodeChallenge13-WhiteboardSS.png)
```

Comment:
Looks like you are off to a great start, when you have some time to come back to this I will regrade for you!

Reply:
Thanks!